without this "udevadm trigger" won't make the devices detected in Solaar
until reboot or replug
--- a/rules.d/42-logitech-unify-permissions.rules
+++ b/rules.d/42-logitech-unify-permissions.rules
@@ -4,7 +4,7 @@
 # Allowing users to write to the device is potentially dangerous
 # because they could perform firmware updates.
 
-ACTION != "add", GOTO="solaar_end"
+ACTION != "add|change", GOTO="solaar_end"
 SUBSYSTEM != "hidraw", GOTO="solaar_end"
 
 # USB-connected Logitech receivers and devices
